src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -1656,6 +1656,120 @@ public function get_test_https_status() {
 		return $result;
 	}
 
+	/**
+	 * Tests if Full Page Cache is available.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @return array The test results.
+	 */
+	public function get_test_page_cache() {
+		$description  = '<p>' . __( 'Page caching enhances the speed and performance of your site by saving and serving static pages instead of calling for a page every time a user visits.' ) . '</p>';
+		$description .= '<p>' . __( 'Page caching is detected by looking for an active page caching plugin as well as making three requests to the homepage and looking for one or more of the following HTTP client caching response headers:' ) . '</p>';
```

Comment:
Same here, and in several other places in the PR. Is it "page cache" or "page caching"?

Reply:
Full Page Cache then?

Reply:
Yep, let's go with that throughout.